core/sdk-core/src/main/java/software/amazon/awssdk/core/RequestOverrideConfiguration.java

```diff
@@ -506,6 +511,26 @@ default B putRawQueryParameter(String name, String value) {
         @SdkPreviewApi
         List<SdkPlugin> plugins();
 
+        /**
+         * Sets the progress listeners used to perform request prorgess tracking
```

Comment:
if possible can we add more java doc , also point to  @ProgressListener class

Reply:
Yes sure I will add more javadoc

Reply:
done